omero/developers/Web/CreateApp.txt

```diff
@@ -358,6 +358,40 @@ of links at the top of the webclient main pages.
        $ bin/omero config append omero.web.ui.top_links '["Homepage", "http://myhome.com", {"title": "Homepage", "target": "new"}]'
 
 
+Custom image viewer
+^^^^^^^^^^^^^^^^^^^
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:`omeroweb.webclient.views.image_viewer`.
```

Comment:
Okay, this property link still isn't working - according to the text added by the autogen build from your code PR, the property is `omero.web.viewer.view` and `omeroweb.webclient.views.image_viewer` is the default setting instead.